com/metacube/EAD/Assignment 3/src/com/database/DataAccessObject.java

```diff
@@ -0,0 +1,219 @@
+package com.database;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException;
+import com.pojo.Employee;
+
+public class DataAccessObject {
+	
+	private static Connection getConnection()
+    {
+        Connection connection = null;
+        
+        try
+        {
+            Class.forName("com.mysql.jdbc.Driver");
+            String url = "jdbc:mysql://localhost:3306/employee";
+            connection = DriverManager.getConnection(url, "root", "adminchirag");
+        }
```

Comment:
Create a label class in utility folder and access username and password from there.